src/cc/frontends/clang/b_frontend_action.cc

```diff
@@ -92,19 +92,17 @@ using namespace clang;
 
 class ProbeChecker : public RecursiveASTVisitor<ProbeChecker> {
  public:
-  explicit ProbeChecker(Expr *arg, const set<tuple<Decl *, int>> &ptregs,
+  explicit ProbeChecker(Expr *arg, const set<tuple<Decl *, int>> &extregs,
                         bool track_helpers, bool is_assign)
-      : needs_probe_(false), is_transitive_(false), ptregs_(ptregs),
+      : needs_probe_(false), extregs_(extregs),
         track_helpers_(track_helpers), nb_derefs_(0), is_assign_(is_assign) {
     if (arg) {
       TraverseStmt(arg);
-      if (arg->getType()->isPointerType())
-        is_transitive_ = needs_probe_;
```

Comment:
This is one of the actual changes (i.e., not renaming). Instead of checking the expression is of pointer type here, we now check the result from that expression has size 64 bits, in the code below (where `is_transitive` used to be called).